cli/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "subsembly-cli",
```

Comment:
Once we publish the library, it would be best to publish it as `subsembly` and not `subsembly-cli`.
Let's rename the name of the package.json to `subsembly`. We could use `subsembly-framework` in the `subsembly` package json (outer scope)